java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -253,6 +254,37 @@ public void set_missing_value_and_returnOldValue_is_null(BaseClient client) {
         assertNull(data);
     }
 
+    @SneakyThrows
```

Comment:
same in here, add transaction tests, I will fix the node wrapper

https://github.com/aws/glide-for-redis/blob/5e41f524e44d40a3d3821db719d6a54c4e7c3f85/python/python/tests/test_transaction.py#L64